src/elements/List/ListItem.js

```diff
@@ -1,54 +1,54 @@
-import React, { Component, PropTypes } from 'react'
 import cx from 'classnames'
+import React, { PropTypes } from 'react'
 
 import {
   customPropTypes,
   getElementType,
+  getUnhandledProps,
   META,
+  useKeyOnly,
 } from '../../lib'
-import { Icon, Image } from '../../elements'
-
-export default class ListItem extends Component {
-  static propTypes = {
-    /** An element type to render as (string or function). */
-    as: customPropTypes.as,
-
-    children: PropTypes.node,
-    className: PropTypes.string,
-    description: PropTypes.node,
-    header: PropTypes.string,
-    icon: PropTypes.node,
-    image: PropTypes.node,
-  }
-
-  static _meta = {
-    name: 'ListItem',
-    type: META.TYPES.ELEMENT,
-    parent: 'List',
-  }
-
-  render() {
-    const { children, className, description, header, icon, image, ...rest } = this.props
-    const classes = cx(className, 'item')
-
-    const media = Icon.create(icon) || Image.create(image)
-    const _description = description || children
-
-    let content = header ? [
-      header && <div key='header' className='header'>{header}</div>,
-      _description && <div key='description' className='description'>{_description}</div>,
-    ] : (
-      _description
-    )
-
-    // wrap content for icon/image layouts
-    if (media) content = <div className='content'>{content}</div>
-    const ElementType = getElementType(ListItem, this.props)
-    return (
-      <ElementType {...rest} className={classes}>
-        {media}
-        {content}
-      </ElementType>
-    )
-  }
+
+function ListItem(props) {
+  const {
+    active,
+    children,
+    className,
+    value,
+  } = props
+  const classes = cx(
+    useKeyOnly(active, 'active'),
+    className,
+    'item'
+  )
+
+  const ElementType = getElementType(ListItem, props)
+  const rest = getUnhandledProps(ListItem, props)
+  const valueProp = ElementType === 'li' ? { value } : { dataValue: value }
+
+  return <ElementType {...rest} {...valueProp} className={classes}>{children}</ElementType>
```

Comment:
I see any React warnings when adding `value` to an `<li>`.  Any reason not to with this?
```jsx
<List.Item value='*' />
```

If there is an issue, one alternative could be to use `bullet`:
```jsx
<List.Item bullet='#' />
```

Though, I'd lean toward SUI parity if we can.